src/clients/c++/CMakeLists.txt

```diff
@@ -29,7 +29,8 @@ cmake_minimum_required (VERSION 3.5)
 #
 # librequest.so and librequest_static.a
 #
-find_package(CURL REQUIRED)
+find_package(CURL CONFIG REQUIRED)
```

Comment:
Since we are building libcurl, should we just remove libcurl installation in Dockerfile and update the documentation?

Reply:
libcurl is needed by GCS and perhaps ONNX. Maybe we could change those builds to instead use our version of libcurl, but that doesn't seem worth the effort.

Reply:
But for the client container we should be able to remove libcurl-dev. I will try.